### PR TITLE
Vocab document Symbols in ISO: https://github.com/metanorma/metanorma…

### PR DESCRIPTION
**Pull request by opoudjis**
…-iso/issues/583

### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

File: author/iso/topics/markup.adoc [hunk 1]
```diff
@@ -298,16 +298,29 @@ and definitions`:
...
+Alternatively they can be listed in an annex:
+
+[source,adoc]
+----
+:docsubtype: vocabulary
+
+....
+
+[appendix]
+== {blank}
```
Comment (ronaldtse) [inline author/iso/topics/markup.adoc:318]: @opoudjis why do we need a blank appendix?

File: author/topics/document-format/xrefs.adoc [hunk 2]
```diff
@@ -88,6 +88,16 @@ Colons in cross-references are used for
 link:/author/topics/document-format/collections#indirect-xrefs[indirect cross-references between files in the same collection],
 to delimit namespaces and containers from anchor IDs [added in https://github.com/metanorma/metanorma-standoc/releases/tag/v1.7.4].
 
+If an anchor is not assigned to an entity, AsciiDoc by default assigns GUIDs to be
```
Comment (ronaldtse) [inline author/topics/document-format/xrefs.adoc:91]: What does GUIDs have to do with AsciiDoc?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: author/iso/topics/markup.adoc (hunk 1 above)
Comment (opoudjis) [reply author/iso/topics/markup.adoc:318]: We don't, really. I just don't know what a symbols clause within an appendix would be called.

File: author/topics/document-format/xrefs.adoc (hunk 2 above)
Comment (opoudjis) [inline author/topics/document-format/xrefs.adoc:91]:
```suggestion
If an anchor is not assigned to an entity, Metanorma by default assigns GUIDs to be
```

File: author/iso/topics/markup.adoc (hunk 1 above)
Comment (ronaldtse) [reply author/iso/topics/markup.adoc:318]: "Symbols and abbreviated terms". I'll fix this.

File: author/topics/document-format/xrefs.adoc [hunk 3]
```diff
@@ -88,6 +88,16 @@ Colons in cross-references are used for
...
+its identifier, prefixed with `_`; e.g. `_01234567-89ab-cdef-0123-456789abcdef`.
+In order to keep anchors stable within a single version of the document,
+Metanorma replaces such GUIDs with a GUID calculated through an MD5 Hash of
```
Comment (opoudjis) [inline author/topics/document-format/xrefs.adoc:94]:
```suggestion
Metanorma calculates those GUIDs as an MD5 Hash of
```

**Pull request merged**